image: amsterdam_total.png
resolution: 0.5
origin: [ -10000, -12500, 0.0]
occupied_thresh: 0.65
free_thresh: 0.196
negate: 0
map_left: 622457
map_right: 632457 
map_bottom: 5798303
map_top: 5810803



